src/client/components/Graph.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import {
+  CartesianGrid,
+  Legend,
+  Line,
+  LineChart,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis
+} from 'recharts'
+
+const graphTitle = {
+  marginLeft: '7rem'
+}
+
+interface GraphProps {
+  title: string
+  data: any[]
+  xDataKey: string
+  labelValue: string
+  lineDataKey1: string
+  lineDataKey2: string
+  lineDataKey3: string
+  lineDataKey4: string
+  color1: string
+  color2: string
+  color3: string
+  color4: string
+  name1: string
+  name2: string
+  name3: string
+  name4: string
+}
+
+export default function Graph(props: GraphProps): JSX.Element {
+  return (
+    <>
+      <h1 style={graphTitle}>{props.title}</h1>
+      <ResponsiveContainer width="95%" height={400}>
+        <LineChart
+          margin={{
+            top: 20,
+            right: 20,
+            bottom: 0,
+            left: 20
+          }}
+          data={props.data}
+        >
+          <CartesianGrid stroke="#f5f5f5" />
+          <XAxis dataKey={props.xDataKey} />
+          <YAxis
+            yAxisId="left"
+            orientation="left"
+            stroke="#000000"
+            width={105}
+            label={{
+              value: props.labelValue,
+              angle: -90,
+              position: 'center'
+            }}
+          />
+          <Tooltip />
+          <Legend />
+          <Line
+            dot={false}
+            legendType="square"
+            yAxisId="left"
+            dataKey={props.lineDataKey1}
+            stroke={props.color1}
+            name={props.name1}
+          />
+          <Line
+            dot={false}
+            legendType="square"
+            yAxisId="left"
+            dataKey={props.lineDataKey2}
+            stroke={props.color2}
+            name={props.name2}
+          />
+          <Line
+            dot={false}
+            legendType="square"
+            yAxisId="left"
+            dataKey={props.lineDataKey3}
+            stroke={props.color3}
+            name={props.name3}
+          />
+          <Line
+            dot={false}
+            legendType="square"
+            yAxisId="left"
+            dataKey={props.lineDataKey4}
+            stroke={props.color4}
+            name={props.name4}
+          />
```

Comment:
With an array of the line information, you can then do an Array.map to create as many lines as needed.